client/widgets/org-stats/stats.template.html.slim

```diff
@@ -0,0 +1,49 @@
+.org-stats.panel.panel-default
+  .panel-heading
+    help-button
+      p Help will be there
+    h3.panel-title Organization DB stats
+  .panel-body
```

Comment:
This widget currently displays nothing when no organization is pre-selected. It should display the organization select component, and a message prompting the user to select and organization: "Select an organization to view its DB stats."